java/ql/src/semmle/code/FileSystem.qll

```diff
@@ -146,9 +146,11 @@ class Container extends @container, Top {
   }
 
   /**
-   * Gets a textual representation of the path of this container.
+   * Gets a textual representation of this container.
    *
-   * This is the absolute path of the container.
+   * The default implementation returns the absolute path to the container, but subclasses may
```

Comment:
```suggestion
   * The default implementation returns the absolute path of the container, but subclasses may
```
The previous text also said "path **of** this container", though maybe "to the" is also fine.

Reply:
As but maybe "returns" needs to be rephrased (e.g. "has ... as result") to conform to the [QLDoc style guide](https://github.com/github/codeql/blob/main/docs/qldoc-style-guide.md#predicates-with-result)?